JASP-Desktop/analysis/analysis.cpp

```diff
@@ -380,8 +380,11 @@ void Analysis::optionsChangedHandler(Option *option)
 	if (_refreshBlocked)
 		return;
 
-	if (form() && form()->hasError())
-		return;
+	if (form())
```

Comment:
Why not `if(form() && (form()->hasError() ||  !form()->runAnalysisWhenThisOptionIsChanged(option))) `, it saves like three lines

Reply:
Done